api.jsx

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const ncNewsAPI = axios.create({
+  baseURL: `https://northcoders-news-ykvv.onrender.com/api`,
```

Comment:
Nice work 👍

Reply:
Thanks and I will heed those comments...Good luck getting through all these Pull Requests!!! 🥵